### PR TITLE
ATM Nano-UI-ification

### PR DESCRIPTION
**Pull request by davipatury**
:cl: davipatury
rscadd: ATM now uses Nano-UI.
/:cl:

<details> 
  <summary>Unauthenticated screen</summary>
  unauthenticated
</details>

<details> 
  <summary>Main screen</summary>
  main
</details>

<details> 
  <summary>Security level screen</summary>
  sec_level
</details>

<details> 
  <summary>Transfer funds screen</summary>
  transfer
</details>

<details> 
  <summary>Transaction log screen</summary>
  log
</details>

### CODE REVIEW

File: nano/templates/atm.tmpl [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+-->
+
+<!--
+#define NO_SCREEN					0
+#define CHANGE_SECURITY_LEVEL		1
+#define TRANSFER_FUNDS				2
+#define VIEW_TRANSACTION_LOGS		3
+-->
+
+<head>
```
Comment (tigercat2000) [inline nano/templates/atm.tmpl]: This is improper HTML, templates are inside the body tag, a head tag should not be defined. Just drop the <head> and leave <style>.

**Comment by tigercat2000** [comment]
👌

**Comment by Fox-McCloud** [comment]
Needs pictures.

**Comment by davipatury** [comment]
@Fox-McCloud fuck

**Comment by Markolie** [comment]
I like!

**Comment by gt33** [comment]
now we need goon's nanoui

**Comment by KasparoVy** [comment]
Looks good

**Comment by Markolie** [comment]
Merge posthaste!

File: nano/templates/atm.tmpl [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	{{/if}}
+{{else data.linked_db}}
+	<form name='atm_auth' action='?src={{:data.src}}' method='get'>
+		<input type='hidden' name='src' value='{{:data.src}}'>
+		<input type='hidden' name='choice' value='attempt_auth'>
+		<div class='itemLabel'>Account:</div> 
+		<div class='itemContent'><input type='text' id='account_num' name='account_num' style='width:250px; background-color:white;'></div>
+		
+		<div class='itemLabel'>PIN:</div>
+		<div class='itemContent'><input type='text' id='account_pin' name='account_pin' style='width:250px; background-color:white;'></div>
```
Comment (Crazylemon64) [inline nano/templates/atm.tmpl]: I could swear I left a comment here remarking that these are white-on-white.

**Comment by davipatury** [comment]
Fixed!

**Pull request merged**